### PR TITLE
Build on net6.0

### PR DESCRIPTION
**Pull request by Gillibald**
## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

File: src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj [hunk 1]
```diff
@@ -105,8 +105,8 @@
       <Compile Include="..\Avalonia.Base\Metadata\NullableAttributes.cs" Link="NullableAttributes.cs" />
       <Compile Remove="../Markup/Avalonia.Markup.Xaml.Loader\xamlil.github\**\obj\**\*.cs" />
       <Compile Remove="../Markup/Avalonia.Markup.Xaml.Loader\xamlil.github\src\XamlX\IL\SreTypeSystem.cs" />
-      <PackageReference Include="Mono.Cecil" Version="0.11.2" />
-      <PackageReference Include="Microsoft.Build.Framework" Version="15.1.548" PrivateAssets="All" />
```
Comment (kekekeks) [inline src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj:109]: This change breaks compatibility with older SDK and Visual Studio versions

**Review by kekekeks — CHANGES_REQUESTED** [review]

File: src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj (hunk 1 above)
Comment (kekekeks) [reply src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj:109]: VS2017 uses MSBuild 15, VS2019 uses MSBuild 16
.NET Core 3.1 SDK (which is LTS and is still supported) uses MSBuild 16

I dont see any particular reason why we should break compatibility with those.

Comment (Gillibald) [reply src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj:109]: Adjusted

Comment (kekekeks) [reply src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj:109]: Make sure that ILRepack picks the correct version

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0024332-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Review by kekekeks — APPROVED** [review]

**Pull request merged**